include/mbedtls/mbedtls_config.h

```diff
@@ -1637,7 +1637,8 @@
 *
 * Enable support for RFC 8446 TLS 1.3 early data.
 *
-* Requires: MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_ENABLED or
+* Requires: MBEDTLS_SSL_SESSION_TICKETS and either
+*           MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_ENABLED or
```

Comment:
Nitpick comment.  Bellow might be better. 
```suggestion
* Requires: MBEDTLS_SSL_SESSION_TICKETS 
* Requires: MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_ENABLED or
```

Reply:
Better to align with the exist coding style, see line 2432, 2775, and other lines, etc.